src/app/pages/marketplace/style.scss

```diff
@@ -79,24 +88,64 @@
     flex-direction: column;
   }
 
-  .marketplace-item .components-card__body {
-    flex-grow: 1;
+  .marketplace-item .components-card__media {
+    position: relative;
+    background-color: #DDDBDD;
+
+    img {
+      min-height: 219.43px;
```

Comment:
It looks like the min-height is making it not very responsive friendly. I'm seeing the thumbnail images getting stretched. I'm guessing the min-height is so the layout doesn't shift as images load. I wonder if there's another way we can accomplish that. Or maybe we combine with `object-fit` so the images to stretch.